lib/public/views/Runs/Runs.js

```diff
@@ -212,9 +237,11 @@ export default class Overview extends Observable {
      * @return {void}
      */
     async createRunsExport(content, fileName) {
-        this.getSelectedExportType() == 'CSV' ?
-            createCSVExport(content, `${fileName}.csv`, 'text/csv;charset=utf-8;')
-            : createJSONExport(content, `${fileName}.json`, 'application/json');
+        if (content.length !== 0) {
```

Comment:
It looks like the user can still input some white spaces and pass this check. 
It would be better to check if the content contains a valid list of numbers